lib/assembly-objectfile/object_fileable.rb

```diff
@@ -148,7 +150,117 @@ def exif_mimetype
       @exif_mimetype ||= begin
         check_for_file
         prefer_exif = !Assembly::TRUSTED_MIMETYPES.include?(file_mimetype) # if it's not a "trusted" mimetype and there is exif data; get the mimetype from the exif
-        exif.mimetype if exif && exif.mimetype && prefer_exif
+        exif.mimetype if # require 'checksum-tools'
+# Common behaviors we need for other classes in the gem
```

Comment:
what is this huge comment block, and why is it in the rubocop commit?  It seems like a separate concept

Reply:
Bug in rubocop 0.75.0, seems better when I use 0.75.1